pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -91,6 +91,7 @@ alloy:
   image:
     tag: {{ .AlloyImageTag }}
   {{- end }}
+  stabilityLevel: experimental
```

Comment:
What is the risk for that? (is there one?)

Reply:
Technically, no, it just allows controls on which features can be enabled so things like this https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.operator.scrapeconfigs/  or livedebugging are not enableable if that is a word by default

This is mostly related to support I would think